interop-cats/shared/src/test/scala/zio/interop/catzSpecBase.scala

```diff
@@ -23,59 +26,123 @@ private[zio] trait catzSpecBase
     with TestInstances
     with catzSpecBaseLowPriority {
 
-  type Env = Clock with Console with System with Random
+  def checkAllAsync(name: String, f: Ticker => Laws#RuleSet): Unit =
+    checkAll(name, f(Ticker()))
+
+  def environment(implicit ticker: Ticker): ZEnv = {
```

Comment:
On the JVM you need to create a test `Blocking` service backed by the `ticker`. Otherwise tests involving blocking effects will fail because the `TestContext` won't see them.

Reply:
Damn, I was thinking about that at some point and then I forgot 👍

Reply:
Ah I remember what was the difficulty here - we need version specific sources 🙈